frontend/pluginloader.lua

```diff
@@ -1,11 +1,24 @@
 local lfs = require("libs/libkoreader-lfs")
 local logger = require("logger")
+local _ = require("gettext")
 
 local DEFAULT_PLUGIN_PATH = "plugins"
+
+-- plugin names that were removed and are no longer available.
 local OBSOLETE_PLUGINS = {
     storagestat = true,
     kobolight = true,
 }
+-- deprecated plugins are still available, but show a hint about its deprecation on UI
+local function getMenuTable(plugin)
+    local t = {}
+    t.name = plugin.name
+    t.fullname = string.format("%s%s", plugin.fullname or plugin.name,
+        plugin.deprecated and " (" .. _("deprecated") .. ")" or "")
```

Comment:
Would something like `outdated` be clearer? Deprecated in this sense is computer jargon.